runtime/polkadot/src/lib.rs

```diff
@@ -36,18 +36,12 @@ use primitives::v0::{
 	AccountId, AccountIndex, Balance, BlockNumber, Hash, Nonce, Signature, Moment,
 	ActiveParas, AbridgedCandidateReceipt, SigningContext,
 };
-use sp_runtime::{
-	create_runtime_str, generic, impl_opaque_keys, ModuleId,
-	ApplyExtrinsicResult, KeyTypeId, Percent, Permill, Perbill,
-	transaction_validity::{
-		TransactionValidity, TransactionSource, TransactionPriority,
-	},
-	curve::PiecewiseLinear,
-	traits::{
-		BlakeTwo256, Block as BlockT, OpaqueKeys, ConvertInto, IdentityLookup,
-		Extrinsic as ExtrinsicT, SaturatedConversion, Verify,
-	},
-};
+use sp_runtime::{create_runtime_str, generic, impl_opaque_keys, ModuleId, ApplyExtrinsicResult, KeyTypeId, Percent, Permill, Perbill, transaction_validity::{
+	TransactionValidity, TransactionSource, TransactionPriority,
+}, curve::PiecewiseLinear, traits::{
+	BlakeTwo256, Block as BlockT, OpaqueKeys, ConvertInto, IdentityLookup,
+	Extrinsic as ExtrinsicT, SaturatedConversion, Verify,
+}};
```

Comment:
Improved readability.

```suggestion
use sp_runtime::{
	create_runtime_str, generic, impl_opaque_keys, ModuleId, ApplyExtrinsicResult, KeyTypeId, Percent, Permill, Perbill,
	curve::PiecewiseLinear,
	transaction_validity::{
		TransactionValidity, TransactionSource, TransactionPriority,
	}, 
	traits::{
		BlakeTwo256, OpaqueKeys, ConvertInto, IdentityLookup, SaturatedConversion, Verify,
		Extrinsic as ExtrinsicT, Block as BlockT,
	},
};
```

Reply:
Happy with the changes but this wasn't actively part of the PR, so maybe resubmit as a pure whitespace PR?